### PR TITLE
PHP7.1 compatibility for issue #96

### PR DESCRIPTION
**Pull request by ankitm123**
Fix for PHP 7.1 compatibility #96. Typecasting the variables fixed the issue. Refer to this page for more details:
https://wiki.php.net/rfc/invalid_strings_in_arithmetic

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,7 +22,8 @@ before_script:
   # Deactivate xdebug if we don't do code coverage
   - if [ -z "$COVERAGE" ]; then rm -rfv /home/travis/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini ; fi
   - composer install
-
+  - composer update
```
Comment (PowerKiKi) [inline .travis.yml]: This should not be here, instead composer.lock file should be included in this PR.

File: test [hunk 2]
```diff
@@ -0,0 +1 @@
+running composer update after composer install to fetch v0.8.0
```
Comment (PowerKiKi) [inline test]: This file should not be here at all

**Comment by ankitm123** [comment]
@PowerKiKi, I added the composer.lock file, and removed the other unnecessary code. Thanks for the suggestion.

**Comment by PowerKiKi** [comment]
Thanks !

**Pull request closed without merging**